modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesKernelBoot.java

```diff
@@ -70,6 +70,12 @@
     private static ScheduledExecutorService andesRecoveryTaskScheduler;
 
     /**
+     * Message Expiration worker task
+     */
+    private static MessageExpirationWorker messageExpirationWorker;
```

Comment:
Done message expiration need to work during a database outage or network is partitioned ?